lib/src/execute.rs

```diff
@@ -55,18 +56,21 @@ pub struct ExecuteCtx {
     next_req_id: Arc<AtomicU64>,
     /// The ObjectStore associated with this instance of Viceroy
     object_store: Arc<ObjectStore>,
+    // Whether to enable wasi-nn functionality
+    // wasi_modules: HashSet<WasiModule>,
```

Comment:
Is this meant to be commented out? If so we should remove it

Reply:
Yup, it is unnecessary once experimental module host functions are linked.